solarforecastarbiter/reference_forecasts/main.py

```diff
@@ -61,18 +70,179 @@ def run(site, model, init_time, start, end):
     ...     init_time, start, end)
     """
 
-    *solpos_forecast, resampler, solar_position_calculator = model(
+    *forecast, resampler, solar_position_calculator = model(
         site.latitude, site.longitude, site.elevation,
         init_time, start, end)
 
     if isinstance(site, datamodel.SolarPowerPlant):
         solar_position = solar_position_calculator()
         ac_power = pvmodel.irradiance_to_power(
             site.modeling_parameters, solar_position['apparent_zenith'],
-            solar_position['azimuth'], *solpos_forecast)
+            solar_position['azimuth'], *forecast)
     else:
         ac_power = None
 
     # resample data after power calculation
-    resampled = list(map(resampler, (*solpos_forecast, ac_power)))
+    resampled = list(map(resampler, (*forecast, ac_power)))
     return resampled
+
+
+def run_persistence(observation, forecast, issue_time, index=False):
+    """
+    Run a persistence *forecast* for an *observation*.
+
+    For intraday forecasts, the *index* argument controls if the
+    forecast is constructed using persistence of the measured values
+    (*index = False*) or persistence using clear sky index or AC power
+    index.
+
+    For day ahead forecasts, only persistence of measured values
+    (*index = False*) is supported.
+
+    The persistence *window* is the time over which the persistence
```

Comment:
I think it would help the docstring to first describe how `data_end` is set.  Then `window` would make more sense.